requirements.txt

```diff
@@ -0,0 +1,6 @@
+-e git://github.com/datamade/rapidsms@master#egg=rapidsms
```

Comment:
I prefer this getting github requirements like 

`https://github.com/datamade/rapidsms-decisiontree-app/zipball/master`

as it does not require the user to have git installed on their stystem.

Reply:
Right. Good.